heron/stmgr/src/cpp/manager/stmgr-server.cpp

```diff
@@ -30,37 +30,61 @@
 #include "network/network.h"
 #include "config/helper.h"
 #include "config/heron-internals-config-reader.h"
+#include "metrics/metrics.h"
 
 namespace heron {
 namespace stmgr {
 
 // Num data tuples received from other stream managers
-const sp_string METRIC_DATA_TUPLES_FROM_STMGRS = "__tuples_from_stmgrs";
+const sp_string METRIC_DATA_TUPLES_FROM_STMGRS = "__server/__tuples_from_stmgrs";
```

Comment:
maybe put "__server/" into a variable and use it in metrics:

const sp_string SERVER_SCOPE = "__server/";
....

during registration:
metrics_manager_client_->register_metric(
    SERVER_SCOPE | METRIC_DATA_TUPLES_FROM_STMGRS,
    tuples_from_stmgrs_metrics_);

Not a blocker though.